.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -291,14 +291,22 @@ jobs:
       working-directory: ${{ matrix.workspace }}
       env:
         RUSTC_WRAPPER: sccache
-    - name: "Test"
+    - name: "CTest"
       run: ctest ${{ matrix.ctest_args }} -C Release -T test --output-on-failure --parallel ${{ matrix.cores }}
       working-directory: ${{ matrix.workspace }}/build
       env:
         RUSTC_WRAPPER: sccache
         QT_QPA_PLATFORM: ${{ matrix.qt_qpa_platform }}
         QT_SELECT: qt${{ matrix.qt_version }}
 
+    - name: "Cargo test"
+      run: cargo test --release --all-features --target-dir ${{ matrix.workspace }}/build/cargo/build
+      working-directory: ${{ matrix.workspace }}
```

Comment:
does this need to be `/build` like above or something ? as it's failing to find Qt on macOS / Windows aka vcpkg ...

Reply:
I think it just needs the right `QMAKE`, but I don't really know how to get that out of CMake/Vcpkg :thinking: 

I think #822 does pretty much what I wanted, as it allows us to see the output of cargo test.